app/cars/page.jsx

```diff
@@ -0,0 +1,65 @@
+'use client';
+
+import { useSearchParams } from 'next/navigation';
+import React from 'react';
+import Link from 'next/link';
+import PropTypes from 'prop-types';
+
+import style from './page.module.css';
+
+const ItemContainer = ({ image, name, cost_per_day }) => {
+  return (
+    <div className={style.itemContainer}>
+      <img src={image} alt='car' className={style.itemImage} />
+      <div>
+        <h3>{name}</h3>
+        <p>- €350 deposit upon any car PuchaSet</p>
+        <table className={style.car_table}>
+          <tr>
+            <td>Finance fee</td>
+            <td>€{cost_per_day}</td>
+          </tr>
+          <tr>
+            <td>Option to purchase fee</td>
+            <td>€{cost_per_day * 2}</td>
+          </tr>
+          <tr>
+            <td>Total ammount payable</td>
+            <td>€{cost_per_day * 10}</td>
+          </tr>
+          <tr>
+            <td>Duration</td>
+            <td>10 days</td>
+          </tr>
+        </table>
+        <p>
+          <strong>5.9% APR</strong>Representative
+        </p>
+        <Link href='/home'>Discover more models</Link>
+      </div>
+    </div>
+  );
+};
+
+export default function page() {
+  const searchParams = useSearchParams();
+  const id = searchParams.get('id'); // To execute the fetch operation
+
+  const response = {
+    id: id,
+    image: 'https://i.ibb.co/5KWVYn6/range.png',
+    name: 'Range Rover',
+    description: 'A great choice for traveling with your besties',
+    cost_per_day: '27',
+  };
+
+  return <ItemContainer {...response} />;
+}
```

Comment:
It would be better if you try to get the data from the API instead of hardcoding it, you can do it by saving some data on the database